wee_slack.py

```diff
@@ -2454,6 +2456,46 @@ def scrolled_cb(signal, sig_type, data):
         pass
     return w.WEECHAT_RC_OK
 
+def convert_attributes(text):
+    if not config.convert_attributes: return text
+    matches = {
+        "*": "bold",
+        "_": "italic",
+        "~": "underline"    # TODO: change to unicode strikethrough
+    }
+    # Find any of the characters within ` and make sure they don't match
+    codes = list(re.finditer("(`.+?`)", text))
+    effect = ""
+    def filter_match(match):
+        # Check if within a code range
+        for code in codes:
```

Comment:
Alternatively, you can use ```(?=([^`]*`[^\`]*\`)*[^\`]*$)``` at the end of the regex. This checks to make sure that there are an even number of backticks in the remainder of the string, which is sufficient since backticks only nest to one layer. I'm not sure which is more performant, but also, that regex will become increasingly hard to read. But, if you add the re.VERBOSE flag and comment judiciously, it should be fine.